front-end/src/app/app.component.html

```diff
@@ -1,6 +1,7 @@
 <main class="d-flex flex-nowrap" style="height: fit-content; min-height: 100%;">
     <app-sidebar class="d-flex flex-column flex-shrink-0 p-3 text-bg-dark col-sm-3 col-12 sidebar" style="max-width: 280px; height: 100vh;"></app-sidebar>
     <div class="container p-4">
-        <app-create-ruleset></app-create-ruleset>
+        <router-outlet></router-outlet>
+        <!-- <app-create-ruleset></app-create-ruleset> -->
```

Comment:
Yes, it should be commented out for now since "router-outlet" is taking over for controlling what is rendered on the page. It was left for now to reference what should be there if router stops working for whatever reason. I suggest its removal when the edit page is created.